modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +45,68 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
+          rootCADir: /etc/cacerts
 ----
++
+[NOTE]
+====
```

Comment:
Making this a note means that we've lost the feature where there's a button to copy code or commands to the clipboard.  I think this would be a smoother flow if it was formatted something like:

Step 3 - After creating/changing/deleting the cacert secret, the control plane istiod and gateway pods must be restarted so the changes go into effect. Use the following command to restart the pods: 

oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway)'.

The Operator will automatically recreate the pods after they have been deleted.

Step 4 (this is moved from the next section) - You must restart the bookinfo application pods so that the sidecar proxies pick up the secret changes.
Use the following command to restart the pods: 

oc -n bookinfo delete pods --all

You should see output similar to the following:

pod "details-v1-6cd699df8c-j54nh" deleted
pod "productpage-v1-5ddcb4b84f-mtmf2" deleted
pod "ratings-v1-bdbcc68bc-kmng4" deleted
pod "reviews-v1-754ddd7b6f-lqhsv" deleted
pod "reviews-v2-675679877f-q67r2" deleted
pod "reviews-v3-79d7549c7-c2gjs" deleted

The pods will be automatically recreated.  Verify that the pods are ready with the following command:

oc get pods -n bookinfo